src/script/auth/module/reducer/clientReducer.ts

```diff
@@ -19,24 +19,22 @@
 
 import type {RegisteredClient} from '@wireapp/api-client/lib/client/';
 
-import {AUTH_ACTION, AppActions, CLIENT_ACTION, NOTIFICATION_ACTION} from '../action/creator/';
+import {AUTH_ACTION, AppActions, CLIENT_ACTION} from '../action/creator/';
 
 export interface ClientState {
-  clients: RegisteredClient[];
-  currentClient: RegisteredClient;
-  error: Error;
+  clients: RegisteredClient[] | null;
+  currentClient: RegisteredClient | null;
+  error: Error | null;
   fetching: boolean;
-  hasHistory: boolean;
   isNewClient: boolean;
 }
 
 export const initialClientState: ClientState = {
-  clients: [],
+  clients: null,
```

Comment:
This default value is to be able to detect if clients have been loaded at all or if it's really an empty list of devices